src/main/antora/modules/ROOT/pages/repositories/core-concepts.adoc

```diff
@@ -37,6 +37,11 @@ public interface CrudRepository<T, ID> extends Repository<T, ID> {
 The methods declared in this interface are commonly referred to as CRUD methods.
 `ListCrudRepository` offers equivalent methods, but they return `List` where the `CrudRepository` methods return an `Iterable`.
 
+[IMPORTANT]
+====
+The repository interface implies a few reserved methods like `findById(ID identifier)` that always target the domain types identifier property regardless of its property name. Read more about this in "`xref:repositories/query-methods-details.adoc#repositories.query-methods.reserved-methods[Defining Query Methods]`".
```

Comment:
We need to watch for broken references in case a Data module doesn't ship a `repositories/query-methods-details.adoc` documentation page.

Reply:
is there one you're aware of right off the top of your head?